### PR TITLE
Add supports for multiple `for_item` parameter for trade offer

### PR DESCRIPTION
**Pull request by aoisensi**
I didn't much test but working for me on Vivaldi.

### CODE REVIEW

**Comment by xPaw** [comment]
Does this match how other (if any) extensions support this param?

File: scripts/community/tradeoffer_new.js [hunk 1]
```diff
@@ -35,7 +40,7 @@ if( item !== null )
 	const element = document.createElement( 'script' );
 	element.id = 'steamdb_for_item';
 	element.type = 'text/javascript';
-	element.appendChild( document.createTextNode( '(' + InjectScript.toString() + '(' + item[ 1 ] + ',' + item[ 2 ] + ',' + item[ 3 ] + '))' ) );
+	element.appendChild( document.createTextNode( '(' + InjectScript.toString() + '("' + items.join( ';' ) + '"))' ) );
```
Comment (xPaw) [inline scripts/community/tradeoffer_new.js:43]: This probably should be `JSON.stringify` instead.

**Comment by aoisensi** [comment]
I haven't seen any other third party tool support multiple parameters.
This is a modification I added because I needed it.

For reference, this user script informs multiple items separated by semicolons.
(However, this script would be different because it uses `them` as a parameter and only supports trading cards.)

**Comment by aoisensi** [comment]
I haven't seen any other third party tool support multiple parameters.
This is a modification I added because I needed it.

For reference, [this user script](https://www.steamtradematcher.com/userscript) informs multiple items separated by semicolons.
(However, this script would be different because it uses `them` as a parameter and only supports trading cards.)

**Comment by xPaw** [comment]
Can you add a little documentation in readme with examples? That certainly would help with adoption.

File: scripts/community/tradeoffer_new.js (hunk 1 above)
Comment (xPaw) [reply scripts/community/tradeoffer_new.js:43]: Now that I think about, the injected script can just read the url, no?

If that works, could you please move the injected script into a separate file so it doesn't need to be stringified, which will align it to how other scripts are injected, see https://github.com/SteamDatabase/BrowserExtension/blob/7fd4f07bca8cff0d1099b34857ae42dbb691889e/manifest.json#L50

**Comment by aoisensi** [comment]
Sorry. I don't want this, because I don't have good English...
It works like this `https://steamcommunity.com/tradeoffer/new/?partner=xxxxxxxxx&token=xxxxxxx&for_item=440_2_xxxxxxxxxx&for_item=440_2_xxxxxxxxxx`

File: scripts/community/tradeoffer_new.js (hunk 1 above)
Comment (aoisensi) [reply scripts/community/tradeoffer_new.js:43]: I thought about it too, but decided not to make too many changes.

**Comment by aoisensi** [comment]
Thank you!

**Pull request closed without merging**